lib/yelp/version.rb

```diff
@@ -1,3 +1,3 @@
 module Yelp
-  VERSION = "2.1.4"
+  VERSION = "1.0.0"
```

Comment:
Not a good standard for handling this. We can move to our own version history on the fork (done here) or use metadata. I opted for a new version history since metadata is not used in determining order.

Reply:
We can also build it and push it to package cloud